packages/terra-aggregator/src/terra-dev-site/test/aggregator/AggregatorWithDisclosure.test.jsx

```diff
@@ -23,19 +24,48 @@ const items = Object.freeze([{
   component: <AggregatorItem name="Section 3" targetId="section3" />,
 }]);
 
-const Wrapper = withDisclosureManager(({ disclosureManager }) => (
+const flippedItems = Object.freeze([{
+  key: 'SECTION_4',
+  component: <AggregatorItem name="Section 4" targetId="section4" />,
+}, {
+  key: 'SECTION_5',
+  component: <AggregatorItem name="Section 5" targetId="section5" />,
+}, {
+  key: 'SECTION_6',
+  component: <AggregatorItem name="Section 6" targetId="section6" />,
+}, {
+  key: 'SECTION_7',
+  component: <AggregatorItem name="Section 7" targetId="section7" />,
+}]);
+
+const Wrapper = withDisclosureManager(({ itemsList, disclosureManager }) => (
   <Aggregator
-    items={items}
+    items={itemsList}
     disclose={disclosureManager.disclose}
   />
 ));
 
-const AggregatorWithDisclosure = () => (
-  <div id="test-aggregator" role="main" className={cx('aggregator-with-disclosure-test')}>
-    <SlidePanelManager>
-      <Wrapper />
-    </SlidePanelManager>
-  </div>
-);
+class AggregatorWithDisclosure extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
Want to try and implement the example using hooks? No pressure, but it'd shrink this example back down a bit.

Reply:
Implemente here c0a69e9625a61825b7aec1215decefc2bfff1a8d